src/ledger-app.js

```diff
@@ -33,6 +33,7 @@ const INS_GET_VERSION = 0x00;
 const INS_PUBLIC_KEY_SECP256K1 = 0x01;
```

Comment:
do we need this anymore ?

Reply:
I just kept it for backward compatibility. Once voyager upgrades, we can remove it

Reply:
we should also inform any third parties so they are aware of the change

Reply:
ok let's create an issue then